example-project/Program.cs

```diff
@@ -8,7 +8,10 @@
 
 // Create logger
 Log.Logger = new LoggerConfiguration()
-    .WriteTo.BetterStack(sourceToken: "SOURCE_TOKEN")
+    .WriteTo.BetterStack(
+        sourceToken: "<source_token>",
+        betterStackEndpoint: "<ingesting-host>"
```

Comment:
<img width="1107" alt="image" src="https://github.com/user-attachments/assets/a9024fd5-a863-4ecf-b7a7-1182007f9619" />

Let's call it "Ingestion host" as it is in UI, and use consistent format 🙏 

```suggestion
        sourceToken: "<source-token>",
        betterStackEndpoint: "<ingestion-host>"
```